README.md

```diff
@@ -1,5 +1,6 @@
-Purescript Syntax/Indentation
-=============================
+# Purescript-vim
+
+> purescript language support in vim and neovim 
```

Comment:
`>` denotes a `<blockquote>` in Markdown and is rendered as such.

> The blockquote element represents a section that is quoted from another source.

— W3C HTML spec, https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element

This block was not not quoting a source so it is not semantic.